### PR TITLE
GH actions for minikube

### PR DESCRIPTION
**Pull request by 0verflowme**

### CODE REVIEW

File: .github/workflows/ci-minikbe.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+#       - name: Checking the logs
+#         run: kubectl logs $(minikube kubectl -- get pods,svc -A | grep pod/kvmsoperator | cut -d ' ' -f 4 | cut -d '/' -f 2) --namespace kube-system
+        
+      - name: Exposing host to machine
+        run : minikube tunnel &
+      
+      - name: Running karmor
+        run: |
+          wget https://github.com//kubearmor/kubearmor-client/releases/download/v0.3/karmor_0.3_linux_amd64.tar.gz
```
Comment (nyrahul) [inline .github/workflows/ci-minikbe.yml:66]: lets use the latest karmor. Lets not hardcode to 0.3 ver

File: .github/workflows/ci-minikbe.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+          sudo mv kubectl /usr/bin
+          sudo chmod 755 /usr/bin/kubectl
+      
+      - name: Start Minikube
+        run: minikube start
+      
+      - name: Apply VM and Hostpolicy CRD
+        run: |
+          sleep 30
+          kubectl apply -f https://raw.githubusercontent.com/kubearmor/kvm-service/dev/deployments/etcd.yml
```
Comment (nyrahul) [inline .github/workflows/ci-minikbe.yml:43]: use the branch on which you are on

File: .github/workflows/ci-minikbe.yml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+#         run: kubectl logs $(minikube kubectl -- get pods,svc -A | grep pod/kvmsoperator | cut -d ' ' -f 4 | cut -d '/' -f 2) --namespace kube-system
+        
+      - name: Exposing host to machine
+        run : minikube tunnel &
+      
+      - name: Running karmor
+        run: |
+          wget https://github.com//kubearmor/kubearmor-client/releases/download/v0.3/karmor_0.3_linux_amd64.tar.gz
+          tar -xvf karmor_0.3_linux_amd64.tar.gz
+          sleep 30
```
Comment (nyrahul) [inline .github/workflows/ci-minikbe.yml:68]: why sleep for 30s?

File: .github/workflows/ci-minikbe.yml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+          curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+          sudo mv kubectl /usr/bin
+          sudo chmod 755 /usr/bin/kubectl
+      
+      - name: Start Minikube
+        run: minikube start
+      
+      - name: Apply VM and Hostpolicy CRD
+        run: |
+          sleep 30
```
Comment (nyrahul) [inline .github/workflows/ci-minikbe.yml:42]: why sleep 30?

**Review by nyrahul — CHANGES_REQUESTED** [review]

File: .github/workflows/ci-minikbe.yml (hunk 4 above)
Comment (0verflowme) [reply .github/workflows/ci-minikbe.yml:42]: it need some time to start correctly , or it will fail

**Pull request closed without merging**